samples/audio/sof/sample.yaml

```diff
@@ -7,5 +7,6 @@ tests:
   sample.audio.sof:
     tags: sof
     build_only: true
+    platform_allow: intel_adsp_cavs15
```

Comment:
This should add all the other cavs platforms that are supported in `samples/audio/sof/boards`